src/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { Env } from 'src/env/index';
+import { JwtPayload } from './jwt-interface';
+import { RedisService } from '@liaoliaots/nestjs-redis';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy, 'jwt') {
+  constructor(
+    private readonly redisService: RedisService,
```

Comment:
PullProofAI

Consider adding type annotations for parameters and return types in the constructor and methods for better code readability and maintainability.

```diff
+   constructor(private readonly redisService: RedisService) {
```



<!-- This is an auto-generated comment by PullProofAI -->